### DISCUSSION TITLE
What is the principle of Quadrature sampler for Polynomial Chaos

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#26825] [Q&A Tools] [answered]
Are there any papers or literature that provide a detailed introduction to this method？

**Comment by GiudGiud** [comment]
@somu15

**Comment by somu15** [comment, marked as answer]
Polynomial chaos requires evaluating integral numerically which can become very expensive when the number of input parameters increases. So, quadrature sampler in STM enables both regular and sparse grid approaches to perform this integration, more efficiently using sparse grids. See this Sandia presentation which looked informative to me: https://www.osti.gov/servlets/purl/1237887. Also, here is our quadrature sampler documentation: https://mooseframework.inl.gov/source/samplers/QuadratureSampler.html.
